### DISCUSSION TITLE
Ground items: notifications for despawning items

### DISCUSSION THREAD

**Discussion by sthoresen** [runelite/runelite#16518] [Ideas]
I am safespotting black dragons and trying to be as afk as possible. I have an idle notification that triggers whenever I finish a dragon, but I have to keep track of the despawning timer for dragon bones myself (I don't want to pick up after every kill, and I want to wait for as long as possible between times I pick them up).

If we could have the option of getting a notification for any time a ground item reaches a despawn time of less than [adjustable in seconds], that would be fantastic. And it would be useful in other content as well. Like afk slayer .

Should be disabled by default as most people probably pay attention and might not have a sophisticated highlighted/hidden items list.

**Comment by Niaxor** [comment]
Second this. Fantastic for any afk monster that you want to keep loot from. 

Trying to get 77 prayer from Green Dragons right now and this would be a god send.
